keycloak/keycloak-operator
# OLM 16.1.0

## Related GH Issue
Closes #425 

## Additional Information
New 16.1.0 of the Keycloak needs a new Operator

## Test Steps
Deploy / Upgrade the operator from the Openshift Marketplace ( test catalog )
Check that it starts and the console opens fine.

## Checklist:
